CONTRIBUTING.md

```diff
@@ -53,30 +61,32 @@ squash.
 
 Test your changes by running:
 
-```console
-$ make lint
+```shell
+make lint
 ```
 
 And you can run the test suite if you have access to elevated permissions:
 
-```console
-# make testunit
-# make integration  # or, if you don't want to use Docker, localintegration
+```shell
+make testunit
+make integration  # or, if you don't want to use Docker, localintegration
```

Comment:
FWIW, I prefer this style, however, I've had several people comment against it and have seen this flip back and forth over time for the same doc on other projects.  To me, it's like putting Ketchup on a hot dog.  Either you like it, or you hate it!